Increase the maximal overall growth of the compilation unit caused
by inlining for dl-reloc.c on hppa. This remove some warnings and 
strangely it reduces the size of the final binary.

---
 ports/sysdeps/hppa/Makefile |    1 +
 1 file changed, 1 insertion(+)

--- a/ports/sysdeps/hppa/Makefile
+++ b/ports/sysdeps/hppa/Makefile
@@ -24,6 +24,7 @@
 
 ifeq ($(subdir),elf)
 CFLAGS-rtld.c += -mdisable-fpregs
+CFLAGS-dl-reloc.c += --param inline-unit-growth=100
 sysdep-dl-routines += dl-symaddr dl-fptr
 sysdep_routines += $(sysdep-dl-routines)
 sysdep-rtld-routines += $(sysdep-dl-routines)
